### PR TITLE
Bb data generation

### PR DESCRIPTION
**Pull request by Barisos22**
Seperated dataframe configuration from data generation, improved wand logging in sweep.py, currently low validation accuracy

### CODE REVIEW

File: best_model_acc.h5
Comment (kenfus) [inline best_model_acc.h5:1]: I think models should not be pushed here but pushed to wandb. However, if you like it like this, I am ok with it. I think at least they should have a versioning.

File: configured_noburst.xlsx
Comment (kenfus) [inline configured_noburst.xlsx:1]: I think this should not be pushed but generated based on the images, on the fly based on parameters.

File: main.py [hunk 1]
```diff
@@ -57,15 +61,21 @@ def main(config_name):
...
+    print("before predict")
     # Calculate other things
     y_pred_proba = model.predict(test_ds).flatten()
+    print("predict-1")
     y_pred = np.where(y_pred_proba > 0.5, 1, 0)
-    y_true = np.concatenate([y for x, y in test_ds], axis=0).flatten()
+    print("predict-2")
+    steps = len(test_ds)  # This will give the number of batches in the test_ds
+    y_true = np.concatenate([y for x, y in islice(test_ds, steps)], axis=0).flatten()
+    print("after predict")
```
Comment (kenfus) [inline main.py:78]: I think those print statements can go, right?

File: sweep.py [hunk 2]
```diff
@@ -33,33 +37,83 @@ def main(config_name: str) -> None:
...
+        monitor="val_accuracy",
+        mode="max",
+        verbose=1,
+        save_best_only=True,
+        save_format="tf",
+    )
+
+    # Create a checkpoint for max val_recall
+    checkpoint_recall = WandbModelCheckpoint(
+        "models/best_model_recall.keras",
```
Comment (kenfus) [inline sweep.py:72]: Ah, now I see the model! Maybe then add models/* to git ignore?

**Review by kenfus — CHANGES_REQUESTED** [review]

File: best_model_acc.h5
Comment (Barisos22) [reply best_model_acc.h5:1]: Saving them locally is optional of course

File: configured_noburst.xlsx
Comment (Barisos22) [reply configured_noburst.xlsx:1]: Yes

File: main.py (hunk 1 above)
Comment (Barisos22) [reply main.py:78]: Yes, sory

File: sweep.py (hunk 2 above)
Comment (Barisos22) [reply sweep.py:72]: Even though these checkpoints are here, they are not actually implemented to the sweep during fitting. They create issues that I haven't been able to fix yet

**Review by kenfus — APPROVED** [review]

**Pull request merged**